Yoast/wordpress-seo
# Introduce notification to upgrade the Person

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Shows a notification to direct the user to the configuration wizard to select a Person when the publish entity is set to "Person".

## Test instructions

This PR can be tested by following these steps:

* Make sure the publishing entity is set to "Person"
* Make sure no user has been selected yet (wpseo_titles > company_or_person_user_id = empty or false)
* See a notification being added to the dashboard
* Select a user (by following the link to the configuration wizard page)
* See the notification no longer being shown

## UI changes

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo/issues/12569
